scipy/interpolate/_cubic.py

```diff
@@ -384,6 +384,11 @@ class Akima1DInterpolator(CubicHermiteSpline):
     axis : int, optional
         Axis in the ``y`` array corresponding to the x-coordinate values. Defaults
         to ``axis=0``.
+    modified : bool, optional
+        If ``True``, use the modified Akima interpolation[1991].
+        Defaults to ``False``, use the Akima interpolation[1970].
```

Comment:
I think some additional document is needed, perhaps in the notes section, explain what the difference is and why you should consider using one or the other

Reply:
I agreed, but I couldn't come up with a good sentence. 😢

Reply:
I wrote it based on references. Please let me know if there are any better expressions.